docs/RELEASE_PROCESS.md

```diff
@@ -1,17 +1,13 @@
 ### `cosmos/cosmos-sdk` Release Process
 
-- [ ] 1. Decide on release designation (are we doing a patch, or minor version bump) and start a branch for the release
-- [ ] 2. Add commits/PRs that are desired for this release **that haven’t already been added to develop**
-- [ ] 3. Merge items in `PENDING.md` into the `CHANGELOG.md`. While doing this make sure that each entry contains links to issues/PRs for each item
-- [ ] 4. Summarize breaking API changes section under “Breaking Changes” section to the `CHANGELOG.md` to bring attention to any breaking API changes that affect RPC consumers.
-- [ ] 5. Tag the commit `git tag -a { .Release.Name }-rcN -m 'Release { .Release.Name }'`
-- [ ] 6. Open PRs for both `master` and `develop`. From now onwards ***no additional PRs targeting develop should be merged**. Additional commits can be added on top of the release branch though.
-- [ ] 7. Ensure both `master` and `develop` PRs ***pass tests***.
-- [ ] 8. Kick off 1 day of automated fuzz testing
-- [ ] 9. Release Lead assigns 2 people to perform [buddy testing script](/docs/RELEASE_TEST_SCRIPT.md) and update the relevant documentation
-- [ ] 10. If errors are found in either #6 or #7 go back to #2 (*NOTE*: be sure to increment the `rcN`)
-- [ ] 11. After #6 and #7 have successfully completed then merge the release PR create the final release annotated tag:
- - `git tag -a -m { .Release.Name } 'Release { .Release.Name }'
- - Merge **the release tag** to both `master` and `develop` so that both branches sit on top of the same commit: `branches='master develop' ; for b in $branches ; do git checkout $b ; git merge { .Release.Name } ; git push $b`.
- Alternatively merge both the aforementioned release PRs.
- - Push the final annotated release tag: `git push --tags`
+Unless otherwise stated, all of the following tasks will be undertaken by the release manager/coordinator:
+
+- [ ] 1. Decide on release designation (are we doing a patch, or minor version bump)
+- [ ] 2. Ensure that all commits/PRs which are destined for this release are merged to the `master` branch
+- [ ] 3. Create the release candidate branch (going forward known as **RC**) and ensure it's protected against pushing from anyone except the release manager/coordinator. **no PRs targeting this branch should be merged unless exceptional circumstances arise**
+- [ ] 4. On the `RC` branch, use `clog` to prepare the `CHANGELOG.md`.
+- [ ] 5. Summarize breaking API changes section under “Breaking Changes” section to the `CHANGELOG.md` to bring attention to any breaking API changes that affect RPC consumers.
+- [ ] 6. Kick off a large round of simulation testing (e.g. 400 seeds for 2k blocks) 
+- [ ] 7. If errors are found in either #6 or #7 go back to #2 (*NOTE*: be sure to increment the `rcN`)
```

Comment:
> (*NOTE*: be sure to increment the `rcN`)

Would this still be required?

Reply:
Should be since further fixes or resolutions needs to be made.

Reply:
There is no mention of `rcN` above, i.e. what is it that should be incremented? A lightweight tag? If so, why should we use it at all? This happens on a separate **RC** branch anyway.

Reply:
It is (kind of) in step (3). No, nothing to do with tags. You create an entirely new branch off of the previous RC branch.

e.g.

1. Create a `release/v0.35.0-rc1` branch off of `master`
2. Run simulation; it fails and there is a bug (ughh ohhh)
3. Create a `release/v0.35.0-rc2` branch off of `release/v0.35.0-rc1`
4. Keep repeating this process until everything looks OK and then you can finally create `release/v0.35.0`

Reply:
> You create an entirely new branch off of the previous RC branch.

I see, that makes sense then.